pkg/consts/consts.go

```diff
@@ -64,12 +64,14 @@ const (
 	// MaxEvents is the maximum number of events we can parse in a single batch.
 	MaxEvents = 5_000
 
+	InngestEventDataPrefix = "_inngest"
```

Comment:
We already seemed to need two variables here, so wanted to roll them up. Either this or the `inngest/function.invoked` event namespaces `data` instead of us adding some `_inngest*` keys to it.